content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
+
+Kita menyebut kasus-kasus yang tidak dapat dihindari sebagai *disrupsi yang tidak disengaja* terhadap aplikasi. Beberapa contohnya adalah sebagai berikut:
+
+- Kesalahahan perangkat keras pada mesin yang menjalankan Node
+- Administrator kluster menghapus *virtual machine* secara tidak sengaja
+- Kesalahan pada penyedia layanan cloud yang mengakibatkan terhapusnya *virtual machine*
+- Sebuah *kernel panic*
+- Node menghilang dari kluster karena partisi jaringan kluster
+- Diusirnya Pod karena Node [kehabisan sumber daya](/docs/tasks/administer-cluster/out-of-resource)
```

Comment:
I think _eviction_ is more like a keyword here not sure if we should translate it or not.
Another choice is:
Pod berada dalam kondisi _evicted_ karena Node..